olxutils/archive.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os
+import shutil
+import tempfile
+
+
+class ArchiveHelper(object):
+    """Helper class to facilitate the creation of course archives to be
+    imported into Open edX Studio."""
+
+    def __init__(self, root_directory, base_name, format='gztar'):
+        # The only format currently supported by Open edX Studio is
+        # gztar, i.e. a gzip-compressed tarball. If that ever changes,
+        # we can easily support others (via shutil.make_archive()).
+        self.base_name = base_name
+        self.root_directory = root_directory
+        self.format = format
+
+    def copy_files(self, destdir):
+        # We currently don't functionally distinguish between files
+        # and directories that are essential, and those that are
+        # (probably?) not.
+        directories = [
+            # apparently essential:
+            'about',
+            'chapter',
+            'html',
+            'info',
+            'policies',
+            'sequential',
+            'static',
+            # apparently not essential:
```

Comment:
I discovered a new one just yesterday: drafts.  (It's where unpushished content lives.)

Reply:
Hmmm. Do we ever want to include drafts in an import? Well we probably do. I'll add it.